lib/tasks/whitehall_importer.rb

```diff
@@ -11,12 +11,13 @@ def initialize(whitehall_document_id, whitehall_document)
       @whitehall_document_id = whitehall_document_id
       @whitehall_document = whitehall_document
       @whitehall_import = store_json_blob
+      @user_map = []
```

Comment:
I find the map part of this variable name a bit confusing as I'd anticipate a map to be an associative array/hash object rather than an array. Might be better to just call it users?